modules/rules_jvm_external/6.1/MODULE.bazel

```diff
@@ -0,0 +1,873 @@
+module(
+    name = "rules_jvm_external",
+    bazel_compatibility = [">=7.0.0"],
+    compatibility_level = 2,
```

Comment:
This changed from 6.0, is that intentional? It could end up being pretty breaking.

Reply:
The bazel compatibility is deliberate. I need to check why the `compatibility_level` changed